test/e2e/upgrade_clusterctl_test.go

```diff
@@ -250,6 +250,8 @@ func preInitFunc(clusterProxy framework.ClusterProxy) {
 		}
 	})
 
+	Byf("Checking out BMO to %s", bmoRelease)
```

Comment:
```suggestion
	By("Checking out BMO to %s", bmoRelease)
```

Reply:
If I want to format `bmoRelease` to the message I have to use`Byf`.